app/models/like.rb

```diff
@@ -1,2 +1,20 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_likes_counter_on_create
+  after_destroy :update_likes_counter_on_destroy
+
```

Comment:
- [ ] It would good to add `after_save` to call the `#update_likes_counter` method every time a new post has been saved to database,

```
`after_save :update_likes_counter`
```